libraries/dawg-house/src/main/webapp/resources/scripts/AddStb.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2010 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var AddStb = (function() {
+
+    var module = {};
+
+    var idLength = 12;
+    var defaultRemoteType = "XR2";
+
+    function makeid() {
```

Comment:
Don't generate the id, make it the lowercase alpha numeric values of the given mac address.

eg. if the supplied mac address is 11:22:AA:BC:DE:FF
Then generate the id as:
1122aabcdeff